index.js

```diff
@@ -53,6 +53,12 @@ module.exports = {
     "react/state-in-constructor": "off",
     // This allows static properties to be placed within the class declaration
     "react/static-property-placement": "off",
+    // Already covered with react/no-array-index-key
+    "sonarjs/no-array-index-key": "off",
+    // Already covered with react/no-unknown-property
+    "sonarjs/no-unknown-property": "off",
+    "sonarjs/jsx-no-useless-fragment": "off",
+    "sonarjs/no-unstable-nested-components": "off",
```

Comment:
[Nit] We should similarly comment above these `// Already covered with react/jsx-no-useless-fragment` and `// Already covered with react/no-unstable-nested-components`.

(It looks like these redundant rules were added because Sonarqube added support to report them as code quality violations in their main product.)